src/components/TabletNameWrapper/TabletNameWrapper.tsx

```diff
@@ -11,13 +11,15 @@ import i18n from './i18n';
 
 interface TabletNameWrapperProps {
     tabletId: string | number;
+    followerId?: string | number;
     database?: string;
 }
 
-export function TabletNameWrapper({tabletId, database}: TabletNameWrapperProps) {
+export function TabletNameWrapper({tabletId, followerId, database}: TabletNameWrapperProps) {
     const isUserAllowedToMakeChanges = useIsUserAllowedToMakeChanges();
 
     const tabletPath = getTabletPagePath(tabletId, {database});
```

Comment:
`FollowerId` should be in page path as well

Reply:
added